rviz_common/CMakeLists.txt

```diff
@@ -27,13 +27,27 @@ find_package(tf2 REQUIRED)
 find_package(tf2_geometry_msgs REQUIRED)
 find_package(tf2_ros REQUIRED)
 
-find_package(PkgConfig)
-pkg_check_modules(YAMLCPP yaml-cpp>=0.5 REQUIRED)
-# if(YAMLCPP_FOUND)
-#   add_definitions(-DRVIZ_HAVE_YAMLCPP_05)
-# endif(YAMLCPP_FOUND)
+if(NOT WIN32)
+  find_package(PkgConfig)
+  pkg_check_modules(YAMLCPP yaml-cpp>=0.5 REQUIRED)
+  # if(YAMLCPP_FOUND)
+  #   add_definitions(-DRVIZ_HAVE_YAMLCPP_05)
+  # endif(YAMLCPP_FOUND)
+else()
+  find_path(YAMLCPP_INCLUDE_DIRS NAMES yaml-cpp/yaml.h)
+  find_library(YAMLCPP_LIBRARIES NAMES libyaml-cppmdd libyaml-cppmd)
+endif()
+
+find_library(TINYXML_LIBRARIES NAMES tinyxmld tinyxml)
+
+# TODO(greimela) Remove as soon as boost is not required anymore (in temp files)
+find_package(Boost REQUIRED)
 
 find_package(ASSIMP REQUIRED)
+if(WIN32)
+  set(ASSIMP_LIBRARIES "${ASSIMP_ROOT_DIR}/lib/${ASSIMP_LIBRARIES}.lib")
```

Comment:
This shouldn't be necessary, if we take it in this pr (just to get Windows working again), then I'd like a TODO (and/or an issue) to come back and fix this up.

Reply:
I'll look into it again, maybe I just messed up my setup.

Reply:
I investigated this issue for quite some time but couldnt come up with a fix. 
Added the TODO now.